src/err.h

```diff
@@ -41,12 +41,12 @@
 void errMsgWrap(char *e, const char *format);
 
 /* Transfer an error message from an object to another, wrapping it. */
-#define ErrMsgTransfer(ERRMSG1, ERRMSG2, FORMAT)     \
-    strncpy(ERRMSG2, ERRMSG1, RAFT_ERRMSG_BUF_SIZE); \
+#define ErrMsgTransfer(ERRMSG1, ERRMSG2, FORMAT)    \
+    memcpy(ERRMSG2, ERRMSG1, RAFT_ERRMSG_BUF_SIZE); \
```

Comment:
This might look odd, but it's safe in practice. Both ERRMSG2 and ERRMSG1 are pointers to buffers of RAFT_ERRMSG_BUF_SIZE bytes. Unfortunately Coverity can't figure that out and emits a warning.

Reply:
One alternative might probably be using `strlcpy`, but that's not in the standard lib afaiu.